app/controllers/registrations_controller.rb

```diff
@@ -10,7 +9,7 @@ def create
     @user = User.new(user_params)
     if @user.save
       auto_login(@user)
-      redirect_to(new_card_path, flash: { info: 'User was sucessfully created' })
+      redirect_to(new_card_path, flash: { info: t('user.created') })
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.